scripts/select_appointments.sql

```diff
@@ -1 +1,18 @@
 -- retrieve appointment information
+SELECT p.surname, p.given_name, d.surname, d.given_name
```

Comment:
We're not interested in the doctor's name - we already know it. However, we _are_ interested in the appointment time.